code/monitor_tutorial/CMakeLists.txt

```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 cmake_minimum_required(VERSION 3.8)
-project(monitor_backend_tutorial)
+project(monitor)
```

Comment:
Use a different name
`monitor_tutorial` for example, so it is same as current path